input_templates/terra_workspaces/cohort_mode/cohort_mode_workspace_dashboard.md.tmpl

```diff
@@ -0,0 +1,183 @@
+12. `module08`: Cohort VCF annotations, including functional annotation, allele frequency (AF) annotation and AF annotation with external population callsets;
+
+Additional modules, such as those for filtering and visualization, are under development. They are not included in this workspace at this time, but the source code can be found in the [GATK-SV GitHub repository]([GATK-SV GitHub repository](https://github.com/broadinstitute/gatk-sv)).
+
+The metrics workflows in this workspace (`module01-metrics`, `module02-metrics`, etc.) are provided for testing purposes and do not need to be executed.
+
+For detailed instructions on running the pipeline, see **Step-by-step instructions** below.
+
+### How many samples can I process at once?
+
+#### Single-sample vs. single-batch vs. multi-batch mode
+
+There are three modes for this pipeline according to the number of samples you need to process:
+
+1. Single-sample mode (<100 samples): The cohort mode of this pipeline requires at least 100 samples, so for smaller sets of samples we recommend the single-sample version of this pipeline, which is available as a [featured Terra workspace](https://app.terra.bio/#workspaces/help-gatk/GATK-Structural-Variants-Single-Sample).
+2. Single-batch mode (100-500 samples)
+3. Multi-batch mode (>200 samples): Batches should be 100-500 samples, so you may choose to divide your cohort into multiple batches if you have at least 200 samples. **The default configuration of this workspace is for multiple batches. Please refer to the step-by-step instructions for details on what to change if you only have one batch.**
+
+
+#### What is the maximum number of samples the pipeline can handle?
+
+In Terra, we have tested batch sizes of up to 500 samples and cohort sizes of up to 1000 samples. On a separate cromwell server, we have tested the pipeline on cohorts of up to ~140,000 samples, but Terra's metadata handling will likely limit cohort sizes further.
+
+
+### Time and cost estimates
+
+The following estimates pertain to the 1000 Genomes sample data in this workspace. They represent aggregated run time and cost across modules for the whole pipeline. For workflows run multiple times (on each sample or on each batch), the longest individual runtime was used. Call caching may affect some of this information.
+
+|Number of samples|Time|Total run cost|Per-sample run cost|
+|--------------|--------|----------|----------|
+|312|~76 hours|~$675|~$2.16/sample|
+
+
+## Running GATK-SV on your data in Terra
+This section will cover how to run the pipeline on your own data in Terra.
+
+### Sample ID requirements 
+
+Sample IDs should be unique and contain only alphanumeric characters and underscores (no dashes, whitespace, or special characters). The same requirements apply to family IDs in the PED file, batch IDs (`sample_set_id`), and the cohort ID (`sample_set_set_id`).
```

Comment:
We should also advise against IDs:
- with all numeric characters
- that are substrings of other IDs
- containing substrings: `chr`, `name`, `DEL`, `DUP`, `CPX`, `CHROM`